book/src/getting-started/3-exposing-to-qml.md

```diff
@@ -26,16 +26,22 @@ There are two notable changes compared to a normal Qt application though:
 {{#include ../../../examples/qml_minimal/cpp/main.cpp:book_qml_register}}
 ```
 
-For every QObject subclass that we define in Rust, CXX-Qt will generate a corresponding C++ class.
-This class is included by the first code snippet.
-They will always be in the `cxx-qt-gen/include/` include path and use the snake_case naming convention.
+For every `#[cxx_qt::bridge]` that we define in Rust, CXX-Qt will generate a corresponding C++ header file.
+This file is included by the first code snippet.
+They will always be in the `cxx-qt-gen/` include path and use the snake_case naming convention.
+The name of the header file will be the name of the Rust module of your `#[cxx_qt::bridge]`, followed by `.cxxqt.h`.
+So in our case: `my_object.cxxqt.h`
 
-The second code snippet then exports the class to QML.
+The second code snippet then exports our `MyObject` class to QML.
 This works the same as it would for any other QObject subclass, as that is exactly what `MyObject` is, as far as Qt is concerned.
```

Comment:
Could also note here, how this is literally a normal C++ class, so you could always derive from it or use it in C++ ways as normal.

Reply:
Done